sqlmesh/core/audit/definition.py

```diff
@@ -29,39 +31,49 @@
 
 if t.TYPE_CHECKING:
     from sqlmesh.core.snapshot import Node, Snapshot
+    from sqlmesh.utils.jinja import MacroReference
 
 if sys.version_info >= (3, 9):
     from typing import Literal
 else:
     from typing_extensions import Literal
 
 
-class AuditMixin:
+class AuditMeta:
```

Comment:
We should probably call this `Mixin` as well. For example `ModelMeta` assumes that we can parse the model definition into it before we even get to complete `Model`. This is not the case here.